user_guides/jaxws/src/main/jbake/content/config.inc

```diff
@@ -43,8 +43,8 @@ slashes as a path separator instead.
   {TechnologyVersion} software is installed
   c.  `PATH` to include the following directories: `JAVA_HOME/bin`,
   +{TechnologyHomeEnv}/bin+, and `ANT_HOME/bin`
-2.  Edit your `<TS_HOME>/bin/ts.jte` file and set the following
-environment variables:
+2.  If you use Jakarta EE 9 Compatible Platform, edit your `<TS_HOME>/bin/ts.jte`
```

Comment:
You could replace the Jakarta EE version ID with {JakartaEEVersion}

Reply:
Yes, done.